docker-compose.yml

```diff
@@ -52,27 +52,35 @@ services:
     env_file:
       - .env
     healthcheck:
-      test: "CMD echo 'SELECT 1' | PGPASSWORD=password psql --host 127.0.0.1 --username postgres --dbname hyrax_development --quiet --no-align --tuples-only"
+      test: "CMD echo 'SELECT 1' | PGPASSWORD=${POSTGRES_PASSWORD} psql --host 127.0.0.1 --username ${POSTGRES_USER} --dbname hyrax_production --quiet --no-align --tuples-only"
       interval: 30s
       timeout: 5s
       retries: 3
     networks:
       internal:
 
-  app:
-    build: hyrax
+  app: &app
```

Comment:
ooh, merges. Do we need to specify compose minor version (since it only appeared in 3.4?) 
https://docs.docker.com/compose/compose-file/#extension-fields

Reply:
I think this is a feature of yml files and not really docker-compose extension fields. Since they dropped support for extends in version 3, looking at SO, people have been using the yml file convention for doing extensions. The fields don't start with x- as mentioned in the docker docs for extension fields